lib/apis/fetch.js

```diff
@@ -8,9 +8,8 @@ export default (url, options = {}) => {
       method: 'GET',
       timeout: config.REQUEST_TIMEOUT_MS,
     });
-
     request(url, opts, (err, response) => {
-      if (!!err || response.statusCode !== 200) {
+      if (!!err || (response.statusCode !== 200 && response.statusCode !== 201)) {
```

Comment:
Should this maybe just be `response.statusCode >= 200 && response.statusCode < 300` ?